askup/utils/views.py

```diff
@@ -253,14 +253,43 @@ def validate_answer_form_and_create(form, request, question):
             question_id=question.id,
             user_id=user.id
         )
-        response['qset_id'] = question.qset_id
-        response['answer_id'] = answer.id
+        response['evaluation_urls'] = get_evaluation_urls(request, question.qset_id, answer.id)
     else:
         response['result'] = 'error'
 
     return response
 
 
+def get_evaluation_urls(request, qset_id, answer_id):
+    """
+    Get the evaluation urls to send to the client side in the question answer form.
+    """
+    filter_value = request.GET.get('filter', '')
+
+    if filter_value:
+        query_args_string = '?filter={}'.format(clean_filter_parameter(filter_value))
+    else:
+        query_args_string = ''
+
+    urls = []
+
+    for evaluation in range(2, -1, -1):
```

Comment:
MAgic numbers?! This is not the best choice it is very easy to get confused with them and make an error char variables will be the more appropriate solution.